Header header

int16 faces
int16 masks